src/client/java/teammates/client/scripts/PlaceholderUserScript.java

```diff
@@ -0,0 +1,14 @@
+package teammates.client.scripts;
+
+/**
+ * Placeholder Script to configure execScript task when not run by user.
+ */
+public final class PlaceholderUserScript {
+    private PlaceholderUserScript() {
+
+    }
+
+    public static void main(String[] args) {
+        System.out.println("Placeholder script has executed");
```

Comment:
Is this message meant to be more informative to the user about the missing argument?

Reply:
Yup thats right